test/suite/test_search_near01.py

```diff
@@ -93,18 +93,19 @@ def test_base_scenario(self):
         cursor2.set_key('aa')
         cursor2.search_near()
 
-        skip_count = self.get_stat(stat.conn.cursor_next_skip_lt_100)
+        skip_count = self.get_stat(stat.conn.cursor_next_skip_total)
         # This should be equal to roughly key_count * 2 as we're going to traverse the whole
         # range forward, and then the whole range backwards.
-        self.assertGreater(skip_count, key_count * 2)
+        self.assertEqual(skip_count, key_count - 1)
```

Comment:
This looks like the test isn't behaving as expected given the original comment do we know why?

Reply:
The test is now using the `cursor_next_skip_total ` statistic, which will skip over only key_count - 1 entries since we evicted the whole range of keys. The previous statistic was incorrectly counting next calls that didn't skip any entries as well (meaning all the next calls used to traverse forwards and backwards), which is why the prior logic of the test was working. I will change the comment here.